base/server/python/pki/server/__init__.py

```diff
@@ -1597,14 +1601,18 @@ def get_token_password(self, token=pki.nssdb.INTERNAL_TOKEN_NAME):
             self.passwords[name] = password
             return password
 
-        except subprocess.CalledProcessError:
-            logger.info('Password unavailable in Keyring.')
+        except (subprocess.CalledProcessError, ValueError) as e:
+            logger.info('Password unavailable in Keyring:', e)
```

Comment:
This too: https://github.com/dogtagpki/pki/actions/runs/18980689068/job/54215665425#step:6:53